cluster-autoscaler/cloudprovider/oci/OWNERS

```diff
@@ -1,5 +1,15 @@
 approvers:
 - jlamillan
+- trungng92
```

Comment:
I think we should be more restrictive with the number of approvers. Also, there is a _requirement_ that approvers (and reviewers) be [Kubernetes org members](https://github.com/kubernetes/community/blob/master/community-membership.md). There is also an expectation that approvers be involved with the autoscaling project (e.g. monitoring for provider issues, regularly attending Kubernetes SIG-Autoscaling, etc.).

Reply:
We (SIG-Autoscaling) don't apply any necessary contribution count threshold to become a reviewer/approver of company-specific code. As long as there is some indication that a person is employed at the company, we are fine with adding them, trusting that the company internal org structure will handle the rest. So we are fine with both long and short list here. Anyway, as @jlamillan is the current approver, he has the final say in this topic.

Reply:
Discussed with Jesse Millan, I've added me and sohan-oracle to the list of reviewers/approvers (commented out)